extensions/data-plane/data-plane-gcs/src/main/java/org/eclipse/dataspaceconnector/gcp/dataplane/gcs/validation/GcsSinkDataAddressValidationRule.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2022 T-Systems International GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       T-Systems International GmbH
+ *
+ */
+
+package org.eclipse.dataspaceconnector.gcp.dataplane.gcs.validation;
+
+import org.eclipse.dataspaceconnector.spi.result.Result;
+import org.eclipse.dataspaceconnector.spi.types.domain.DataAddress;
+
+import java.util.List;
+
+import static org.eclipse.dataspaceconnector.gcp.core.storage.GcsStoreSchema.BUCKET_NAME;
+
+public class GcsSinkDataAddressValidationRule implements ValidationRule<DataAddress> {
+
+    @Override
+    public Result<Void> apply(DataAddress dataAddress) {
+        var composite = new CompositeValidationRule<>(
```

Comment:
this could be a class field, and it does not need to be a "composite" since there's only one rule (`EmptyValueValidationRule`)